test/functional/wallet_resendwallettransactions.py

```diff
@@ -40,6 +44,7 @@ def run_test(self):
 
         self.log.info("Create a new transaction and wait until it's broadcast")
         txid = int(node.sendtoaddress(node.getnewaddress(), 1), 16)
+        wtxid = int(node.getrawtransaction(hex(txid)[2:], 1)['hash'], 16)
```

Comment:
This is dropping the padding left zeroes I think with `hex(txid)[2:]`.

Just save txid above as a string, then convert to int after this line and use it directly, or:
```
"{0:0{1}x}".format(txid, 64)
```
(I think former is easier :P )

Reply:
How interesting. Must be a python version thing. I needed to do this locally to strip off the leading `0x`. Will fix.

Reply:
Sorry I mean for example the txid is like "5". It's going to just be `0x05` or something rather than `0x00000000....05`

Reply:
This will happen if the txid is a sufficiently small `int`

Reply:
got it. So if the leading byte is 0x00 this fails (1/256 times)

Reply:
Fixed